.prettierignore

```diff
@@ -0,0 +1,2 @@
+yarn.lock
+package-lock.json
```

Comment:
Nice. Haha we aren't supposed to have both of these lockfiles, so I will submit a small PR to remove the `package-lock.json`.